src/time.ts

```diff
@@ -17,18 +14,18 @@ export interface HardhatTimeHelpers {
   lastBlockTime(): Promise<number>
   /**
    * Increases block timestamp by the specified time period.
-   * @param {BigNumberish} time Time period that should pass to the next mined block.
+   * @param {bigint} time Time period that should pass to the next mined block.
    * @return {number} Timestamp of the next block.
    */
-  increaseTime(time: BigNumberish): Promise<BigNumber>
+  increaseTime(time: number): Promise<number>
```

Comment:
`bigint` or `number`?

Reply:
after refactoring let's use `number` - same as in the hardhat's implementation